src/signTx.c

```diff
@@ -781,6 +664,7 @@ static void signTx_handleTtl_ui_runStep()
 
 	UI_STEP(HANDLE_TTL_STEP_DISPLAY) {
 		char ttlString[50];
+		// TODO just display it as a plain uint64, the same as validity interval start
```

Comment:
as we discussed yesterday, I'd keep the TTL formatting logic and perhaps generalize the name to `str_formatValidityBoundary` or something like that